rbd/group.go

```diff
@@ -135,7 +137,8 @@ func GroupImageRemove(groupIoctx *rados.IOContext, groupName string,
 		cephIoctx(groupIoctx),
 		cGroupName,
 		cephIoctx(imageIoctx),
-		cImageName)
+		cImageName,
+		C.uint32_t(0))
```

Comment:
This change is part of the [in-progress ceph PR](https://github.com/ceph/ceph/pull/53793/files) that has mirror group APIs